src/content/docs/en/guides/fonts.mdx

```diff
@@ -8,23 +8,17 @@ i18nReady: true
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
 
 
-
-Astro supports all common strategies for adding custom typefaces to your site design. This guide will show you two different options for including web fonts in your project.
+This guide will show you how to add web fonts to your project and use them in your components.
 
 ## Using a local font file
 
-If you want to add font files to your project, we recommend adding them to your [`public/` directory](/en/core-concepts/project-structure/#public). In your CSS you can then register fonts with an [`@font-face` statement](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face) and use the `font-family` property to style your site.
-
-### Example
-
-Let’s imagine you have a `DistantGalaxy.woff` font file.
+Let's imagine you have a `DistantGalaxy.woff` font file.
 
 1. Add your font file to `public/fonts/`.
-
-2. Add an `@font-face` statement to your CSS. This could be in a global `.css` file you import or in a `<style>` block in the layout or component where you want to use this font.
+1. Add an `@font-face` statement to your CSS. This could be in a global `.css` file you import, a `<style is:global>` block, or a `<style>` block in a specific layout or component where you want to use this font.
```

Comment:
```suggestion
1. Add the following `@font-face` statement to your CSS. This could be in a global `.css` file you import, a `<style is:global>` block, or a `<style>` block in a specific layout or component where you want to use this font.
```
This allows us to avoid the "a/an" question, because a reader might not read the "at" out loud.